packages/dapp/src/components/SettingsCard/TrustedDappTable.tsx

```diff
@@ -9,7 +22,50 @@ import InputField from '../InputField';
 
 export const TrustedDappTable = () => {
   const t = useTranslations('TrustedDappTable');
+  const { isOpen, onOpen, onOpenChange } = useDisclosure();
   const [settings, setSettings] = useState<MascaConfig>();
+  const [loading, setLoading] = useState<boolean>(false);
+  const [permissions, setPermissions] = useState<
+    Record<string, DappPermissions>
+  >({});
+  const [currentDapp, setCurrentDapp] = useState<any>({
+    permissions: {
```

Comment:
We have `any` type here which is fine, but we probably don't need this big object. We have it in 2 places and has potential to get out of sync.